addons/upgrade/to-13.0-convert-hpswitch-types.pl

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/perl
```

Comment:
Please rename the file from addons/upgrade/to-13.0-convert-hpswitch-types.pl to addons/upgrade/to-13.1-convert-hpswitch-types.pl